### PR TITLE
[1822direkt] Add broker with activities for: buy, sell (without taxes), dividends (from fonds)

### LINKED ISSUE #58: 1822direkt Support
Weiter unten sind die einzelnen JSONs + Screenshots der PDFs damit man es leichter zuordnen kann.  
Soweit ich weiß fehlt nur noch ein PDF für eine Dividende in EUR.

PDF-Typen:
 * Kauf Börse
 * Kauf Direkthandel
 * Sparplankauf
 * Verkauf
 * Dividende USD

### PR DESCRIPTION
**Pull request by dredav**
* Fix #58
* Add tests with samples from #58
* Add broker support for 1822direkt

Still missing is the support for:
* Buys with taxes
* Sells with taxes
* Dividends from stocks

### CODE REVIEW

**Comment by dredav** [comment]
I've updated the [community post](https://community.tresor.one/c/help/der-automatische-import-benotigt-deine-hilfe-um-noch-besser-zu-werden) for 1822direkt:
>1822direkt: Abrechnungen von Aktien Dividenden, Käufte/Verkäufe mit gezahlten Steuern

File: tests/brokers/__mocks__/1822direkt/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  require('./dividend/etf110_without_taxes.json'),
+];
+
+export const buySamples = [
+  require('./buy/market_order_direct.json'),
+  require('./buy/market_order_exchange.json'),
+  require('./buy/saving_plan.json'),
+];
+
+export const sellSamples = [require('./sell/market_order.json')];
```
Comment (carstenhag) [inline tests/brokers/__mocks__/1822direkt/index.js:20]: Also turn into 3 lines

File: tests/brokers/1822direkt.test.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      allSamples.forEach(samples => {
+        expect(samples.some(item => _1822direkt.canParseData(item))).toEqual(
+          true
+        );
+      });
+    });
+
+    test('Can identify a broker from one page as 1822direkt', () => {
+      allSamples.forEach(samples => {
+        expect(samples.some(item => getBroker(item) === _1822direkt)).toEqual(
```
Comment (carstenhag) [inline tests/brokers/1822direkt.test.js:24]: I liked this part more at the scalable PR  :D
```
        expect(
          samples.some(item => scalableCapital.canParseData(item))
        ).toEqual(true);
```

**Comment by dredav** [comment]
@carstenhag Thanks for reviewing. Both cases addresses as the comment from [here](https://github.com/codeofsumit/tresor-import/pull/90#discussion_r479882007).

**Review by carstenhag — APPROVED** [review]

**Pull request merged**